string operation
geometry_msgs/Point position
float64 outside_radius
float64 inside_radius
float64 weight